Formula/libmongocrypt.rb

```diff
@@ -1,8 +1,8 @@
 class Libmongocrypt < Formula
   desc "C library for Client Side Encryption"
   homepage "https://github.com/mongodb/libmongocrypt"
-  url "https://github.com/mongodb/libmongocrypt/archive/1.2.0.tar.gz"
-  sha256 "0a16c53dc81efef5a32b42cd65c225e89f3eb48f9c45f6f1d5026278b9865c48"
+  url "https://github.com/mongodb/libmongocrypt/archive/1.2.1.tar.gz"
+  sha256 "5b76aa92f07267dab0169154fd7b4fd1302f50b0d2498155c57179e351dda7e5"
   license "Apache-2.0"
   head "https://github.com/mongodb/libmongocrypt.git"
 
```

Comment:
Does there need to be an update on line 17 (maybe 15 too) for `BUILD_VERSION`?

Reply:
@kevinAlbs - Just saw your comment in the other review. Since that one appears more complete, we will just merge that and then close this PR out.

Reply:
Well, I spoke to soon. @deriamis backed out those changes from #119. @kevinAlbs can you add in the missing change for `BUILD_VERSION` and then I'll approve this for merge?